src/measure_theory/interval_integral.lean

```diff
@@ -647,8 +681,141 @@ begin
     simp [h] }
 end
 
+lemma congr_ae' {f g : α → E} (h : ∀ᵐ x ∂μ, x ∈ Ioc a b → f x = g x)
+  (h' : ∀ᵐ x ∂μ, x ∈ Ioc b a → f x = g x) :
+  ∫ (x : α) in a..b, f x ∂μ = ∫ (x : α) in a..b, g x ∂μ :=
+by simp only [interval_integral, set_integral_congr_ae (measurable_set_Ioc) h,
+              set_integral_congr_ae (measurable_set_Ioc) h']
+
+lemma congr_ae {f g : α → E} (h : ∀ᵐ x ∂μ, x ∈ Ι a b → f x = g x) :
+  ∫ (x : α) in a..b, f x ∂μ = ∫ (x : α) in a..b, g x ∂μ :=
+interval_integral.congr_ae' (ae_interval_oc_iff.mp h).1 (ae_interval_oc_iff.mp h).2
+
+lemma integral_zero_ae  {f : α → E} (h : ∀ᵐ x ∂μ, x ∈ Ι a b → f x = 0) :
+  ∫ (x : α) in a..b, f x ∂μ = 0 :=
+calc ∫ x in a..b, f x ∂μ = ∫ x in a..b, 0 ∂μ : interval_integral.congr_ae h
+                     ... = 0                 : interval_integral.integral_zero
+
+lemma integral_indicator {a₁ a₂ a₃ : α} (h : a₂ ∈ Ioc a₁ a₃) {f : α → E} :
+  ∫ x in a₁..a₃, indicator {x | x ≤ a₂} f x ∂ μ = ∫ x in a₁..a₂, f x ∂ μ :=
+begin
+  have : {x | x ≤ a₂} ∩ Ioc a₁ a₃ = Ioc a₁ a₂, from Iic_inter_Ioc_of_le h.2,
+  rw [interval_integral.integral_of_le h.1.le, interval_integral.integral_of_le (h.1.le.trans h.2),
+      integral_indicator, measure.restrict_restrict, this],
+  exact measurable_set_Iic,
+  all_goals { apply measurable_set_Iic},
+end
+
 end order_closed_topology
 
+section continuity_wrt_parameter
+open topological_space
+
+variables {X : Type*} [topological_space X] [first_countable_topology X]
+variables {μ : measure α}
+
+lemma continuous_at_of_dominated_interval
```

Comment:
informal docstring for this one and the next ones?

Reply:
Done